usr/lib/cca_stdll/cca_specific.c

```diff
@@ -5989,6 +6114,7 @@ CK_RV token_specific_aes_cbc(STDLL_TokData_t * tokdata,
     return rc;
 }
 
+
```

Comment:
Minor nit: please don't add this empty line here. You are removing it with the next commit (i.e. you insert a function there).